gtk/gtk.go

```diff
@@ -9065,6 +9066,21 @@ func (v *TreeModelFilter) SetVisibleColumn(column int) {
 	C.gtk_tree_model_filter_set_visible_column(v.native(), C.gint(column))
 }
 
+// ConvertChildPathToPath is a wrapper around gtk_tree_model_filter_convert_child_path_to_path().
+func (v *TreeModelFilter) ConvertChildPathToPath(childPath *TreePath) *TreePath {
+	path := C.gtk_tree_model_filter_convert_child_path_to_path(v.native(), childPath.native())
+	if path == nil {
+		return nil
+	}
+	p := &TreePath{path}
+	return p
+}
```

Comment:
Please check if `childPath` is nil and do an early bail with a second return value of type `error`

also, please use the global standard error for unexpected cgo nil value called `nilPtrErr` and don't return only nil value

-> see `func ButtonNew() (*Button, error)` as reference

additionally:  
this is purely up to you, but maybe you want to write a `wrapTreePath` func (see existing `wrapTreeModel` as reference)